drivers/SmartThings/matter-button/src/init.lua

```diff
@@ -270,8 +275,12 @@ end
 ----------------------------------------------------------------------------
 
 -- initial press
-local function initial_event_handler(driver, device, ib, response)
-  if not get_field_for_endpoint(device, MULTI_BUTTON, ib.endpoint_id) then
+local function initial_press_event_handler(driver, device, ib, response)
+  if get_field_for_endpoint(device, SUPPORTS_MULTI_PRESS, ib.endpoint_id) then
+    -- Receipt of an InitialPress event means we do not want to ignore the next MultiPressComplete event
+    -- or else we would potentially not create the expected button capability event
+    set_field_for_endpoint(device, IGNORE_NEXT_MPC, ib.endpoint_id, nil)
```

Comment:
nit: is there a reason to set this as nil instead of `false`? nil would make me think it hadn't been intentionally set yet.

Reply:
just saves memory. truth value would be the same, though